styles/images.css

```diff
@@ -0,0 +1,13 @@
+.image-res {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+.sq-img {
+  height: 100%;
+  object-fit: cover;
+}
+.round-img {
+  border-radius: 50%;
+  object-fit: cover;
```

Comment:
could have given consistant css names like either img-sq img-round..because above css is img-res

Reply:
done